app/models/transfer.rb

```diff
@@ -32,12 +32,14 @@ def save
       @out_transaction = Transaction.new(
         amount_cents: estimate_amount(out = true),
         bank_account_id: bank_account_id, comment: form_comment(comment),
+        date: date,
```

Comment:
@yunixon before filling date for transaction set it to some value if it blank, so both transaction will have same date.